wysadzulice/tests.py

```diff
@@ -1,3 +1,11 @@
-from django.test import TestCase
+from django.test import TestCase, Client
```

Comment:
W dniu 02.05.2016 o 12:08, Tomasz Magulski pisze:
>> @@ -1,3 +1,11 @@
>> -from django.test import TestCase
>> +from django.test import TestCase, Client
> 
> Nie ma tego nigdzie napisanego (bo w sumie nie było czasu napisać jakiś guideline dla devów), ale preferowałbym się trzymać jednej linii per import (https://www.python.org/dev/peps/pep-0008/#imports), więc tutaj bym zrobił:
> ```
> from django.test import Client
> from django.test import TestCase
> ```
> 
> Trochę więcej pisania, ale mi co najmniej linter nie wariuje na wielu liniach.

Dokładnie w tym samym linku który przytoczyłeś jest, że linter nie ma
prawa wariować, bo to się tyczy tylko wielokrotnych importów różnych
modułów.